README.md

```diff
@@ -1,52 +1,32 @@
-> [!NOTE]
-> Hi MonoGame community, MonoGame.Extended is currently being updated to resolve outstanding bug issues, included those surrounding Tiled, and to update for MonoGame 3.8.1.303.  Please bear with me as I work through the backlog.  You can follow the progress in the [v4.0.0 Milestones](https://github.com/craftworkgames/MonoGame.Extended/milestone/8) page.
```

Comment:
With items still outstanding (including tiled), did you intent to remove this comment and link to milestones?